src/Common/src/Interop/Windows/user32/Interop.GetWindowTextLengthW.cs

```diff
@@ -9,7 +9,7 @@ internal partial class Interop
 {
     internal partial class User32
     {
-        [DllImport(Libraries.User32, EntryPoint = "GetWindowTextLengthW")]
-        public static extern int GetWindowTextLength(IntPtr hWnd);
+        [DllImport(Libraries.User32, SetLastError = true, ExactSpelling = true)]
+        public static extern int GetWindowTextLengthW(IntPtr hWnd);
```

Comment:
It does, but this is a perf thing (albeit small). It avoids the scrounging and saves a string.